libs/shared/src/containers/Account/reducer.ts

```diff
@@ -165,26 +164,19 @@ function AccountReducer(address: string, initialState: AccountState) {
           undefined :
           (per: number) => {
             const percent = Math.floor(per * 100);
-            if (!callback(percent)) {
-              throw ("Operation cancelled");
-            }
+            return !callback(percent);
           }
```

Comment:
Throwing here somehow escapes the catch below, however we could just return the cb value and let ethers throw for us